src/utils/debug/texture-selector-utils.ts

```diff
@@ -4,15 +4,15 @@
 // We need to store tiles when we replace texture
 const tiles = {};
 
-export function selectDebugTextureForTileset(tileset, uvDebugTexture) {
+export async function selectDebugTextureForTileset(tileset, uvDebugTexture) {
```

Comment:
Could you add TSDoc comments and types for the function? It was ported from JS long time ago and types were omited

Reply:
Done